src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -251,25 +251,10 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     getMetadataService().setServiceSceneState(this.state);
     this._subs.unsubscribe();
 
-    this.clearAdHocVariables();
```

Comment:
Is this related with the other changes? Can't see how.

Reply:
If left in, it breaks the ability to "undo" go back to start